ParabolInc/action
# Fix back and forth column dnd

This fixes #1567.

To verify:
- Set up your columns as follows:
```
| [ c ] | [ a ] |
|       | [ b ] |
```
- all in one drag, pull card `a` on top of `c`, and then back to the right where it used to be; where `b` now is.
- the preview card should follow the drag as expected.
- all other dragging and dropping should work as expected!
